docs/using-the-compiler.rst

```diff
@@ -127,12 +127,12 @@ Input Description
         //   evm.ir - New assembly format before desugaring
         //   evm.assembly - New assembly format after desugaring
         //   evm.legacyAssemblyJSON - Old-style assembly format in JSON
-        //   evm.opcodes - Opcodes list
-        //   evm.methodIdentifiers - The list of function hashes
-        //   evm.gasEstimates - Function gas estimates
         //   evm.bytecode - Bytecode
+        //   evm.bytecode.opcodes - Opcodes list
```

Comment:
Probably it makes more sense to only have `evm.bytecode` which turns on `object`, `opcodes`, `sourceMaps` and `linkRerefences.`